m0kv_metadata_parsing_tool/m0kv_metadata_parsing_tool.sh

```diff
@@ -77,80 +65,91 @@ get_probable_delete_index() {
         declare -a PART_LIST_INDEX_OID
         rm /tmp/m0kvprobable.log > /dev/null 2>&1
         cd ..
-        ./"$m0kv_PATH" -l "$1" -h "$2" -p "$3" -f "$4" index next "$probable_delete_index" '0' 1000 -s >> /tmp/m0kvprobable.log
+        "$m0kv_PATH" -l "$1" -h "$2" -p "$3" -f "$4" index next "$probable_delete_index" '0' 1000 -s >> /tmp/m0kvprobable.log
```

Comment:
can we take input from user as to where exactly do we want to save the log ?

Or this log file will be deleted later by the script ?

Reply:
Log file is stored in /var/log/